cime_config/allactive/config_pesall.xml

```diff
@@ -7094,7 +7094,7 @@
       </pes>
     </mach>
   </grid>
-  <grid name="a%ne120np4_">
+  <grid name="a%ne120np4">
```

Comment:
Why are you making this change? I added the underscores here to distinguish between np4 and pg2 grids.

Reply:
Ok, I didn't see that underscore was specifically added in E3SM-Project/E3SM#3204 to disable running pg2 cases with existing PE-layouts.
I was just trying to get some data from old PE-layouts. I added a GH issue E3SM-Project/E3SM#4568 to note the need for new PE-layouts.